lib/options.js

```diff
@@ -4,6 +4,12 @@ module.exports.process = function(opts) {
   // set the default directory to run everything out of
   opts.dir = opts.dir || '/home/codewarrior';
 
+  // opts.code is an alias for opts.solution
+  if (opts.code) {
+    opts.solution = opts.code;
+    delete opts.code;
```

Comment:
We can't have `delete opts.code;` yet because some language runners uses `opts.code`.

The build matrix caught this :)

![image](https://cloud.githubusercontent.com/assets/639336/25419384/211effa6-2a07-11e7-8414-64d07629fc79.png)